c_common/front_end_common_lib/include/tdma_processing.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2017-2019 The University of Manchester
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*! \file
+ *
+ *  \brief local Time Division Multi Access Functions Header File
+ *
+ *    provides function for spending packets within a given time frame
+ *    simulations.
+ *
+ */
+
+#ifndef _TDMA_PROCESSING_H_
+#define _TDMA_PROCESSING_H_
+#include <stdbool.h>
+
+//! stores the format of the TDMA processing state in SDRAM
+typedef struct tdma_parameters {
+    //! The time at which the first message can be sent
+    uint32_t initial_expected_time;
+    //! The time at which the last message must be sent by
+    uint32_t min_expected_time;
+    //! The time between sending
+    uint32_t time_between_sends;
+} tdma_parameters;
+
+//! \brief hands back the number of times the TDMA was behind
+uint32_t tdma_processing_times_behind(void);
+
+//! \brief init for the tdma processing
+//! \param[in/out] address: pointer to the SDRAM address where this data is
```

Comment:
It's `\param[in,out]`; alas, Doxygen is picky about this.

Reply:
fixed